package.json

```diff
@@ -37,6 +37,7 @@
     "he": "^1.2.0",
     "papaparse": "^5.4.1",
     "rxjs": "^7.8.0",
+    "scss": "^0.2.4",
```

Comment:
Does this require any new NPM scripts to generate the actual CSS?

Reply:
In my understanding, the existing setup already handles the SCSS compilation and no additional steps are required.
Just the normal "npm i" is enough to make the project run correctly.
Does this answer your question?